lib/validation.js

```diff
@@ -45,11 +47,12 @@ module.exports = function (yargs, usage) {
         }
       })
 
-      if (missingRequiredArgs.length === 1) {
-        usage.fail('Missing argument value: ' + missingRequiredArgs[0])
-      } else if (missingRequiredArgs.length > 1) {
-        var message = 'Missing argument values: ' + missingRequiredArgs.join(', ')
-        usage.fail(message)
+      if (missingRequiredArgs.length > 0) {
+        usage.fail(__n(
+          'Missing argument value: %s', 'Missing argument values: %s',
```

Comment:
Minor nit: I like how you put the singular and plural strings on their own lines elsewhere (e.g. lines 84-85, lines 116-117) since it helps make the differences much more obvious IMO (even based on alignment alone). I would repeat that pattern here.